bridge/engine/EngineCommand.h

```diff
@@ -241,8 +240,18 @@ union CommandUnion
 
 struct Command
 {
+    Command& operator=(const Command&) = delete;
+    Command& operator=(Command&& rhs)
+    {
+        _type = rhs._type;
+        _command = rhs._command; // TODO would need move assignment in the substructs ?
+        _packet = std::exchange(rhs._packet, nullptr);
```

Comment:
```suggestion
        _packet = std::move(rhs._packet);
```

`std:: exchange` creates a temporary object. Usually is used on move assign operator and move contractor to exchange the values of native types. The other objects that also implements the move assign I think the `std::move` is preferable